### PR TITLE
chore!: v0.9.5

### PR DESCRIPTION
**Pull request by enthusiastmartin**
## Description
v0.9.5

### CODE REVIEW

**Review by Roznovjak — COMMENTED** [review]
LGTM, but I'm not super familiar with the changes in `service.rs`.

File: pallets/exchange/src/tests.rs [hunk 1]
```diff
@@ -177,6 +177,10 @@ fn sell_test_pool_finalization_states() {
 				user_3_sell_intention_id,
 			)
 			.into(),
+			orml_tokens::Event::Reserved(asset_a, 2, 1000000000000).into(),
+			orml_tokens::Event::Reserved(asset_b, 3, 2000000000000).into(),
+			orml_tokens::Event::Reserved(asset_b, 2, 2000000000).into(),
+			orml_tokens::Event::Reserved(asset_b, 3, 4000000000).into(),
```
Comment (green-jay) [inline pallets/exchange/src/tests.rs:183]: was wondering if this repetitive part could be setup and reused as a function?

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -210,6 +210,7 @@ impl Filter<Call> for BaseFilter {
 
 			Call::XYK(_)
 			| Call::AssetRegistry(_)
+			| Call::Balances(_)
```
Comment (green-jay) [inline runtime/src/lib.rs:213]: will staking, governance etc. work?

File: pallets/exchange/src/tests.rs (hunk 1 above)
Comment (enthusiastmartin) [reply pallets/exchange/src/tests.rs:183]: yes.possibly. but i have noticed little discrepancy in this, so i investigate that first. And when all good, we can handle this kind of improvement.

File: runtime/src/lib.rs (hunk 2 above)
Comment (enthusiastmartin) [reply runtime/src/lib.rs:213]: Yes, it should. It was tested previously with treasury tip.

**Review by lumir-mrkva — CHANGES_REQUESTED** [review]
please bump versions and test running main chain node

**Review by lumir-mrkva — APPROVED** [review]

**Pull request merged**